Content.Server/Body/Systems/BloodstreamSystem.cs

```diff
@@ -82,7 +79,8 @@ public override void Update(float frameTime)
     {
         base.Update(frameTime);
 
-        foreach (var bloodstream in EntityManager.EntityQuery<BloodstreamComponent>())
+        var query = EntityQueryEnumerator<BloodstreamComponent>();
+        while (query.MoveNext(out var uid, out var bloodstream))
         {
             bloodstream.AccumulatedFrametime += frameTime;
```

Comment:
You should take a look at this if you haven't seen it used before, pretty neat. Replacing a foreach EntityQuery and component.Owner with a while on an EntityQueryEnumerator, which seems to collect UIDs as well.